src/azure-cli-core/setup.py

```diff
@@ -46,6 +46,7 @@
     'argcomplete~=1.8',
     'azure-cli-telemetry==1.0.6.*',
     'azure-common~=1.1',
+    'azure-core==1.12.0',
```

Comment:
Also do this in `setup.py` so that CLI installed with `pip` will also only use `azure-core==1.12.0`.